runlike/inspector.py

```diff
@@ -90,6 +95,14 @@ def parse_ports(self):
                         '--expose=%s' %
                         container_port_and_protocol)
 
+        exposed_ports = self.get_fact("Config.ExposedPorts")
+        if exposed_ports:
+            for container_port_and_protocol, options in exposed_ports.items():
+                if not ports or container_port_and_protocol not in ports.keys():
```

Comment:
```suggestion
                if container_port_and_protocol not in ports.keys():
```

Reply:
The binding ports and exposed ports may both appear in "Config.ExposedPorts", but only the  binding ports will appear in "NetworkSettings.Ports". So i think the code is correct.